airlock_processor/StatusChangedQueueTrigger/__init__.py

```diff
@@ -136,3 +147,15 @@ def get_source_dest_for_copy(new_status: str, request_type: str, short_workspace
             dest_account_name = constants.STORAGE_ACCOUNT_NAME_EXPORT_BLOCKED + short_workspace_id
 
     return ContainersCopyMetadata(source_account_name, dest_account_name)
+
+
+def report_failure(outputEvent, request_properties, failed_reason):
```

Comment:
Nit: I would use `reason_failed` instead of `failed_reason`

Reply:
I actually prefer `X_reason` as it's more natural, maybe  `failure_reason`  makes more sense